src/components/WishlistCard.jsx

```diff
@@ -0,0 +1,95 @@
+import { Link, useLocation } from "react-router-dom";
```

Comment:
Keep this component in the `components` folder only if it is being used in more than one page / view else keep it inside its own page folder

Reply:
Implemented the suggestion